docs/tabpy-tools.md

```diff
@@ -8,6 +8,7 @@ on TabPy server.
 - [Connecting to TabPy](#connecting-to-tabpy)
 - [Authentication](#authentication)
 - [Deploying a Function](#deploying-a-function)
+- [Predeployed Functions](#predeployed-functions)
```

Comment:
TOC is updated with markdown-toc as described in CONTRIBUTING.md. You shouldn't edit it manually. Also, individual models are not listed in here.

Reply:
I removed my TOC edit. The individual models are listed below Predeployed Functions, they don't need to be listed specifically in the TOC especially if we continue to add more the list will grow very long